README.md

```diff
@@ -273,8 +282,8 @@ Typescript is supported via Javascript and Jsx lexers. If you are using Javascri
   handlebars: [
     {
       lexer: 'HandlebarsLexer',
-      functions: ['t'] // Array of functions to match
-    }
+      functions: ['t'], // Array of functions to match
+    },
```

Comment:
Please make sure you only commit changes relevant to what you worked on. I don't want to merge style changes.

Reply:
> Thanks for the PR, it looks good. My only concern is that I would like more test at the parser level. I don't like the code coverage reduction. Could you add some while you address my other comment about the README?

No problem, I should have an update shortly.

Reply:
Just a heads up, I think all those additional changes were made automatically on commit because of this line: https://github.com/i18next/i18next-parser/blob/master/.lintstagedrc.js#L2

So, if other submitters change the README.md I suspect you'll see the same changes occur.